eventbridge/platform/aws/publish.js

```diff
@@ -0,0 +1,23 @@
+const { EventBridgeClient, PutEventsCommand } = require("@aws-sdk/client-eventbridge");
+
+const putEvent = async (eventBridgeName, event) => {
+  console.log(`Publishing event to EventBridge: ${eventBridgeName} - ${JSON.stringify(event)}`);
+  const client = new EventBridgeClient({ region: process.env.AWS_REGION });
+  const input = {
```

Comment:
I think it makes more sense to put this in the Wing code (minimize the amount of .js code).